main.go

```diff
@@ -229,7 +229,10 @@ func main() {
 			Client:  mgr.GetClient(),
 			Decoder: admission.NewDecoder(mgr.GetScheme()),
 
-			Skipper: psk,
+			Skipper: skipper.NewMultiSkipper(
```

Comment:
Do we really want to fully skip this webhook? Don't we need the ns quota part that's backed by the configmap regardless of whether usageprofiles are enabled or not?